data/sql/updates/pending_db_world/FixMaiZothFrostArmor.sql

```diff
@@ -0,0 +1,7 @@
+
+UPDATE creature_template SET AIName = 'SmartAI' WHERE entry = 818;
```

Comment:
The NPC already has a smart_scripts, so it makes an update, it is not necessary in this case. This is done, when the npc or whatever is being programmed, still has nothing assigned, but if you are deleting by removing the `id`, that means, that at least the `smart_scripts` has 2 more lines, so the update is unnecessary in this case.